ui/src/views/Hooks/ViewHook/index.jsx

```diff
@@ -20,6 +21,9 @@ import exchangesList from '../../../utils/exchangesList';
     variables: {
       hookGroupId: params.hookGroupId,
       hookId: decodeURIComponent(params.hookId),
+      hookConnection: {
+        limit: VIEW_CLIENTS_PAGE_SIZE,
```

Comment:
UI doesn't have a full pagination built in yet, so it will only display first 1000 records.
My reasoning is that we might not need to go beyond that.

Reply:
I think that's fine..